beetsplug/replaygain.py

```diff
@@ -1553,8 +1554,10 @@ def close_pool(self):
     def import_begin(self, session):
         """Handle `import_begin` event -> open pool
         """
-        if self.config['auto']:
-            self.open_pool(self.config['threads'].get(int))
+        threads = self.config['threads'].get(int)
+
+        if self.config['parallel_on_import'] and threads:
```

Comment:
Should this also include `self.config['auto']` to avoid running the pool if we're not supposed to do RG at all?

Reply:
Yes it should, thanks!